docs/java-api/index.asciidoc

```diff
@@ -44,12 +44,12 @@ You need to also include Log4j 2 dependencies:
 <dependency>
     <groupId>org.apache.logging.log4j</groupId>
     <artifactId>log4j-api</artifactId>
-    <version>2.7</version>
+    <version>2.8.2</version>
```

Comment:
Not between docs and outside the docs. The docs are built independently, do not use Gradle, and do not consume the entire repository.